lib/public/views/Logs/LogsModel.js

```diff
@@ -113,7 +113,7 @@ export class LogsModel extends Observable {
      * @param {string} [queryParams.environmentIds] the environment ids to link to the log being created
      * @return {void}
      */
-    loadCreation({ runNumbers, lhcFillNumbers, environmentIds }) {
+    loadCreation({ runNumbers, lhcFillNumbers, environmentIds, templateKey, issueDescription, detectorOrSubsystem }) {
```

Comment:
I think that the template things should be in a separate property. Basically you have the relations of the log, such as related runs, and you also have the template payload (template key and template data).